lib/base/utility.cpp

```diff
@@ -1310,6 +1310,14 @@ bool Utility::PathExists(const String& path)
 	return fs::exists(fs::path(path.Begin(), path.End()), ec) && !ec;
 }
 
+time_t Utility::GetFileCreationTime(const String& path)
+{
+	namespace fs = boost::filesystem;
+	fs::path filePath(path);
```

Comment:
Why a local variable?

Reply:
why should I need a member variable, which is only needed in this function?

Reply:
I didn’t say anything about member variables. I just asked what's this local variable for?

Reply:
Because the function does not accept string paths, but `boost::filesystem::path&` instead.

Reply:
What about casting the string to a path then?

Reply:
Yes, that would work too!